integration/scenario_test.go

```
@@ -502,8 +533,8 @@ func updateElection(secret kyber.Scalar, proxyAddr, electionIDHex, action string
 	if err != nil {
 		return 0, xerrors.Errorf("failed retrieve the decryption from the server: %v", err)
 	}
-
-	require.Equal(t, resp.StatusCode, http.StatusOK, "unexpected status: %s", resp.Status)
+	body, _ := io.ReadAll(resp.Body)
```

never ignore an error